vsintegration/tests/FSharp.Editor.Tests/Hints/HintTestFramework.fs

```diff
@@ -40,9 +41,10 @@ module HintTestFramework =
     let getFsiAndFsDocuments (fsiCode: string) (fsCode: string) =
         RoslynTestHelpers.CreateTwoDocumentSolution("test.fsi", SourceText.From fsiCode, "test.fs", SourceText.From fsCode)
 
-    let getHints document hintKinds =
+    let getHints (document: Document) hintKinds =
         async {
-            let! hints = HintService.getHintsForDocument document hintKinds "test" CancellationToken.None
+            let! sourceText = document.GetTextAsync CancellationToken.None |> Async.AwaitTask
```

Comment:
I know it hasn't been changed in this PR, but curious on why is `CancellationToken.None` is used as oppose to one from `Async`?

Reply:
That's because I didn't know it exists :D updated